### PR TITLE
Adds more missing models

### PR DESCRIPTION
**Pull request by Kim2091**
Added a ton of models again. Please ensure all changes are correct

### CODE REVIEW

File: data/models/3x-DAT-2.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "DAT_2_X3",
+    "author": "zheng-chen",
+    "license": null,
```
[the remark this answers is not in the document]
Comment (joeyballentine) [reply data/models/3x-DAT-2.json:4]: Should not having a license fail the CI?

[the remark this answers is not in the document]
Comment (RunDevelopment) [reply data/models/3x-DAT-2.json:4]: Probably, but we'll have to figure out what to do [with those models](https://openmodeldb.info/?t=license%3Aunknown) before.

[the remark this answers is not in the document]
Comment (Kim2091) [reply data/models/3x-DAT-2.json:4]: @RunDevelopment there was no license associated with the file that I could find

[the remark this answers is not in the document]
Comment (RunDevelopment) [reply data/models/3x-DAT-2.json:4]: The repo is licensed under Apache 2.0, so I would assume the models are too.

[the remark this answers is not in the document]
Comment (Kim2091) [reply data/models/3x-DAT-2.json:4]: Well yes, but assuming isn't ideal when it comes to models, right? It'd be safer to leave it unmarked imo

File: data/models/4x-DWTP-DS-dat2-v3.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    "inputChannels": 3,
+    "outputChannels": 3,
+    "resources": [
+        {
+            "platform": "pytorch",
+            "type": "pth",
+            "size": 140299118,
+            "sha256": "7ce4d37f9b9ef951086e721d79a371342e092904e841ef8cc38484abd8f73050",
+            "urls": [
+                "https://cdn.discordapp.com/attachments/1170374027144089762/1186368034651246672/4x_DWTP_DS_DAT2_V3.pth?ex=65dcd255&is=65ca5d55&hm=07e68a86ba321daf67344818f0f300190a5e851854ec85a312a5557a1d039efc&"
```
[the remark this answers is not in the document]
Comment (Kim2091) [reply data/models/4x-DWTP-DS-dat2-v3.json:29]: I replied to you about this on Discord

[the remark this answers is not in the document]
Comment (RunDevelopment) [reply data/models/4x-DWTP-DS-dat2-v3.json:29]: They seem to be working still, so let's keep them for now.

File: data/models/3x-DAT-2.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (RunDevelopment) [reply data/models/3x-DAT-2.json:4]: https://github.com/zhengchen1999/DAT/issues/30#issuecomment-1959271956

They are Apache 2.0.

**Comment by RunDevelopment** [comment]
Please run `npm run fix-db`.

I made the db check stricter yesterday.

**Review by RunDevelopment — APPROVED** [review]
Thank you @Kim2091!

**Pull request merged**